src/api/entities/Portfolio/index.ts

```diff
@@ -211,9 +209,9 @@ export abstract class Portfolio extends Entity<UniqueIdentifiers, HumanReadable>
   /**
    * Send an invitation to an Identity to assign it as custodian for this Portfolio
    *
-   * @note this will create an AuthorizationRequest which has to be accepted by the corresponding Identity.
-   *   An Account or Identity can fetch its pending Authorization Requests by calling {@link Authorizations.getReceived}.
-   *   Also, an Account or Identity can directly fetch the details of an Authorization Request by calling {@link Authorizations.getOne}
+   * @note this will create an {@link AuthorizationRequest Authorization Request} which has to be accepted by the `targetIdentity`.
```

Comment:
```suggestion
   * @note this will create an {@link AuthorizationRequest | Authorization Request} which has to be accepted by the `targetIdentity`.
```